packages/components/src/DatePicker/_docs/DatePicker.stories.tsx

```diff
@@ -249,3 +250,118 @@ export const Validation: Story = {
 export const DisabledDays: Story = {
   parameters: { controls: { include: /^disabled/ } },
 }
+
+export const LimitedWindowWidth: Story = {
+  name: "At 400% window size",
+  parameters: {
+    controls: { disable: true },
+    viewport: {
+      viewports: {
+        ViewportAt400: {
+          name: "Viewport at 400%",
+          styles: {
+            width: "320px",
+            height: "350px",
+          },
+        },
+      },
+      defaultViewport: "ViewportAt400",
+    },
+    a11y: { disable: true }, // accessible label fix to be addressed in a separate PR
```

Comment:
Funnily enough disabling one cause a cascade of other issues that seem to relate to no title, so figured I'd just plop this on here since it'll be a fast follow fix :)